test_programs/execution_success/unsigned_to_signed_cast/src/main.nr

```diff
@@ -0,0 +1,6 @@
+fn main(x: u64) {
+    assert_eq(x as i8, -1);
+    assert_eq(x as i16, -1);
+    assert_eq(x as i32, -1);
+    assert_eq(x as i64, -1);
+}
```

Comment:
Is this the same program (with the same input value) as `signed_truncation`?

Reply:
Ah, my bad. we should have a `i64` as argument to the other function.

Reply:
@asterite I've addressed this now